### ISSUE TITLE
Unicode Support

### ISSUE THREAD

**Issue by bexelbie** [awdeorio/mailmerge#19]
It would be nice to have unicode support so that users with unicode names could spell them correctly.

I've been looking and it seems we could do this if we always sent a UTF-8 encoded email, easily.

WDYT?

**Comment by awdeorio** [comment]
Unicode email bodies work.  I haven't tried a Unicode sender name.  Do you think you could provide a very small example that fails with the current implementation?  Then, I can take a look at how to make it work.

**Comment by bexelbie** [comment]
see attached.

[mailmerge_template.txt](https://github.com/awdeorio/mailmerge/files/1268086/mailmerge_template.txt)

**Comment by awdeorio** [comment]
I'm a little confused.  It seems to work as expected for me:

```shellsession
$ mailmerge --version
mailmerge, version 1.7.6
$ mailmerge 
>>> message 0
TO: XXX@pobox.com
SUBJECT: Foo
FROM: Brian Exelbierd <XXX@pobox.com>

Hi Kristýna,


>>> sent message 0 DRY RUN
>>> Limit was 1 messages.  To remove the limit, use the --no-limit option.
>>> This was a dry run.  To send messages, use the --no-dry-run option.

**Comment by bexelbie** [comment]
Please try it with some valid email addresses and send an actual message.

**Comment by rupertlevene** [comment]
I am experiencing the same issue with UTF8 text in email bodies: the dry run succeeds, but sending email fails. 

```
$ file mailmerge_template.txt
mailmerge_template.txt: ASCII text
$ file mailmerge_database.csv
mailmerge_database.csv: UTF-8 Unicode text
$ python --version
Python 3.5.2
$ mailmerge --version
mailmerge, version 1.7.7
$ mailmerge
>>> message 0
TO: my.valid.email+test@gmail.com
SUBJECT: Testing mailmerge UTF8 support
FROM: valid.sending@somewhere.com
Content-Type: text/plain; charset="UTF-8"

Hi, Séan


>>> sent message 0 DRY RUN
>>> Limit was 1 messages.  To remove the limit, use the --no-limit option.
>>> This was a dry run.  To send messages, use the --no-dry-run option.
$ mailmerge --no-dry-run
>>> message 0
TO: my.valid.email+test@gmail.com
SUBJECT: Testing mailmerge UTF8 support
FROM: valid.sending@somewhere.com
Content-Type: text/plain; charset="UTF-8"

Hi, Séan


>>> Read SMTP server configuration from mailmerge_server.conf
>>>   host = smtp.gmail.com
>>>   port = 465
>>>   username = valid.sending@somewhere.com
>>>   security = SSL/TLS
>>> password for valid.sending@somewhere.com on smtp.gmail.com:
Traceback (most recent call last):
  File "/mnt/bigstore/Dropbox/morn/eventbrite/mailmerge/env/bin/mailmerge", line 9, in <module>
    load_entry_point('mailmerge==1.7.7', 'console_scripts', 'mailmerge')()
  File "/mnt/bigstore/Dropbox/morn/eventbrite/mailmerge/env/lib/python3.5/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/mnt/bigstore/Dropbox/morn/eventbrite/mailmerge/env/lib/python3.5/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/mnt/bigstore/Dropbox/morn/eventbrite/mailmerge/env/lib/python3.5/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/mnt/bigstore/Dropbox/morn/eventbrite/mailmerge/env/lib/python3.5/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/mnt/bigstore/Dropbox/morn/eventbrite/mailmerge/env/lib/python3.5/site-packages/mailmerge/__main__.py", line 45, in cli
    config_filename=config_filename,
  File "/mnt/bigstore/Dropbox/morn/eventbrite/mailmerge/env/lib/python3.5/site-packages/mailmerge/api.py", line 232, in main
    sendmail(message, config_filename)
  File "/mnt/bigstore/Dropbox/morn/eventbrite/mailmerge/env/lib/python3.5/site-packages/mailmerge/api.py", line 94, in sendmail
    message.as_string(),
  File "/usr/lib/python3.5/smtplib.py", line 850, in sendmail
    msg = _fix_eols(msg).encode('ascii')
UnicodeEncodeError: 'ascii' codec can't encode character '\xe9' in position 160: ordinal not in range(128)
$ cat mailmerge_template.txt
TO: my.valid.email+test@gmail.com
SUBJECT: Testing mailmerge UTF8 support
FROM: valid.sending@somewhere.com
Content-Type: text/plain; charset="UTF-8"

Hi, {{name}}

$ cat mailmerge_database.csv
email,name,number
myself@mydomain.com,"Séan",17
bob@bobdomain.com,"Bob",42
```

**Comment by awdeorio** [comment]
Update: I can reproduce this bug.

**Comment by awdeorio** [comment]
This is now fixes in version 1.7.8
